src/features/search/components/SearchResults/SearchResults.native.test.tsx

```diff
@@ -423,6 +423,19 @@ describe('SearchResults component', () => {
       useFeatureFlagSpy.mockReturnValueOnce(true)
     })
 
+    it('should open the venue modal when pressing the venue filter button', async () => {
+      render(<SearchResults />)
+      const venueButton = screen.getByTestId('venueButtonLabel')
+
+      await act(async () => {
+        fireEvent.press(venueButton)
+      })
+
+      expect(screen.getByTestId('fullscreenModalView')).toHaveTextContent(
+        'Trouver un point de vente'
+      )
```

Comment:
pinaillage :

pour rappel, la structure des tests : AAA :

1. Arrange
2. Act
3. Assert

j'aime sauter une ligne entre chaque étape pour montrer la différence

le venueButton est utile pour agir dessus, je le mettrais donc dans le `act`


```suggestion
    it('should open the venue modal when pressing the venue filter button', async () => {
      render(<SearchResults />)

      await act(async () => {
        const venueButton = screen.getByTestId('venueButtonLabel')
        fireEvent.press(venueButton)
      })

      expect(screen.getByTestId('fullscreenModalView')).toHaveTextContent(
        'Trouver un point de vente'
      )
```

Reply:
Je suis alignée, je corrige, merci !